app/models/region.rb

```diff
@@ -23,36 +25,33 @@ class Region < ActiveRecord::Base
                     s3_credentials: { bucket: 'boulder-food-rescue-robot-region-photo' }
   validates_attachment_file_name :logo, matches: [/png\Z/, /jpe?g\Z/, /gif\Z/]
 
-  def active_volunteer_count
-    volunteers = self.schedule_chains.collect_concat {|schedule| schedule.volunteers }
-    volunteers.uniq.count
+  def self.has_any_handbooks?(region_list)
+    region_list.any?(&:has_handbook?)
   end
 
-  def self.all_admin(volunteer)
-    Region.where(id: volunteer.admin_region_ids)
+  def active_volunteer_count
+   schedule_chains.flat_map(&:volunteers).uniq.count
```

Comment:
I think its important to start marking things as #TODO for when we upgrade if we'll get performance benefits / cleaner code. I'm not sure uniq vs distinct does much here, but I'm all for adding #TODO comments.